### PR TITLE
Assistant Traitor Item: Part Electric Boogatwo

### PR DESCRIPTION
**Pull request by ghost**
Adds a traitor item costing 2 TC, takes the look of the old (not rusty) toolbox.
Does 19 brute and 22 when thrown, carries simple tools and the nearly just as notorious insulated gloves.

It now does more damage per telecrystal. (thank you MrDoomBringer)
Max brute is 37.

:cl: duckay
add: Adds a traitor item for assistants, a odd-looking toolbox that does increased damage every TC that is within it.
imageadd: The ye olde toolbox. (oldtoolboxclean)
:cl:

i can't into github

why: we don't have enough role restricted items for tg's posterboy.

### CODE REVIEW

**Comment by ExcessiveUseOfCobblestone** [comment]
lol

**Comment by duckay** [comment]
syndie toolbox does 15 brute and 18 when thrown this does 18 brute and only costs 4 shekels.

**Comment by MMMiracles** [comment]
why do we need more traitor items designed solely for murdering people and nothing else

**Comment by duckay** [comment]
> why do we need more traitor items designed solely for murdering people and nothing else

because traitors often get objects to murder people and nothing else???

**Comment by duckay** [comment]
oops

**Comment by optimumtact** [comment]
You'd have to rewrite the history of the branch and force push it

you'd want something like
```
on your branch
#rebase interactively the last four commits
git rebase -i HEAD~4
#This will bring up an editor window where you can chose what to do with each of the 
#4 commits
```
example
```
#choose pick for most of them, but the one commit where you made the oopsie
#change the pick to a reword or the letter r, then exit the editor (on the cli this is 
#normally vim) you can exit it via pressing :wq and then <enter>, also press i to
#enter insert mode to type, and <esc> to exit insert mode
#this will let you edit that commit and repick the rest as is
#once done force push your branch to the remote
git push --force --upstream origin <branchname>
```

**Comment by Tlaltecuhtli** [comment]
dupe of syndicate toolbox

**Comment by Kierany9** [comment]
>dupe of syndicate toolbox

this

**Comment by Tlaltecuhtli** [comment]
spray blue paint on the syndicate toolbox so it looks like the old one

**Comment by ExcessiveUseOfCobblestone** [comment]
all this work just to insert the old toolbox sprite in smh.

just toss it into the legacy space shuttle ruin you memer.

**Comment by ATH1909** [comment]
I feel like this should cost less; maybe make it cost the same as a syndicate toolbox?

It's role-exclusive (to assistants, no less), so I think being better than a non-role-exclusive item is okay.

**Comment by duckay** [comment]
Given that it has 4 more brute (15 + 2 brute per TC) it costs 2 TC.

**Comment by subject217** [comment]
oranges it's a web editor pr so they can't force push

also this item is basically an excuse to use the old toolbox sprite and not much else, pretty much the same as the syndie toolbox

File: code/game/objects/items/storage/toolbox.dm [hunk 1]
```diff
@@ -86,6 +86,24 @@
 
 /obj/item/storage/toolbox/mechanical/old/heirloom/PopulateContents()
 	return
+	
+/obj/item/storage/toolbox/mechanical/old/clean
+	name = "mechanical toolbox"
+	desc = "A blue, old toolbox, it looks robust."
```
Comment (coiax) [inline code/game/objects/items/storage/toolbox.dm]: The name and description should be identical to a regular old heirloom toolbox.

**Comment by ExcessiveUseOfCobblestone** [comment]
webedits actually just make a branch on your fork so it's still possible, you just have to do a bit more work than had you started in git ~~desktop~~

File: code/game/objects/items/storage/toolbox.dm (hunk 1 above)
Comment (duckay) [reply code/game/objects/items/storage/toolbox.dm]: Should it really say "It's seen better days" when the toolbox is in pristine condition.
As for the name I'll change that to just 'toolbox'.

Comment (ExcessiveUseOfCobblestone) [reply code/game/objects/items/storage/toolbox.dm]: The sprite has seen better days when it was a common item

Comment (duckay) [reply code/game/objects/items/storage/toolbox.dm]: wouldn't it be better to have a description to indicate it's robust?

Comment (ExcessiveUseOfCobblestone) [reply code/game/objects/items/storage/toolbox.dm]: idc lol it's bikeshed

Comment (duckay) [reply code/game/objects/items/storage/toolbox.dm]: ill just close this review then

**Comment by MrDoomBringer** [comment]
it should do more damage the more raw telecrystals are in it

**Comment by MrDoomBringer** [comment]
that way you can have a big boy beat weapon but if someone takes it than kek ur TC is gone sucko

**Comment by duckay** [comment]
> it should do more damage the more raw telecrystals are in it

dont make me cry

**Comment by nicbn** [comment]
Fix your changelog, no text before cl

**Comment by MrDoomBringer** [comment]
@duckay merge this loser https://github.com/duckay/tgstation/pull/1

**Comment by duckay** [comment]
its done for anyone curious

File: code/game/objects/items/storage/toolbox.dm [hunk 2]
```diff
@@ -86,6 +86,31 @@
...
+	item_state = "toolbox_blue"
+	has_latches = FALSE
+	force = 19
+	throwforce = 22
+
+/obj/item/storage/toolbox/mechanical/old/clean/attack(mob/target, mob/living/user)
+	var/power = 0
+	for (var/obj/item/stack/telecrystal/TC in GetAllContents())
+		power += TC.amount
+	force = 19 + power
```
[the remark this answers is not in the document]
Comment (duckay) [reply code/game/objects/items/storage/toolbox.dm]: i didn't code this, but throwing is for pussies.

[the remark this answers is not in the document]
Comment (PKPenguin321) [reply code/game/objects/items/storage/toolbox.dm]: Opposite tbh

[the remark this answers is not in the document]
Comment (duckay) [reply code/game/objects/items/storage/toolbox.dm]: I'll update it tomorrow when I'm awake then

[the remark this answers is not in the document]
Comment (duckay) [reply code/game/objects/items/storage/toolbox.dm]: gosh you didn't need to lol but it's merged and this is resolved

**Pull request merged**